src/drivers/audio/portaudio/portaudio_info.c

```diff
@@ -126,7 +123,12 @@ const PaDeviceInfo * Pa_GetDeviceInfo(PaDeviceIndex device) {
 }
 
 const PaHostApiInfo * Pa_GetHostApiInfo(PaHostApiIndex hostApi) {
-	return NULL;
+	static const PaHostApiInfo info = {
+		.structVersion = 1,
+		.name = "stm32_audio_host_api",
```

Comment:
If it's placed in common  src/drivers/audio/portaudio/portaudio_info.c file why name is `.name = "stm32_audio_host_api"`